cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/spark/SparkRuntimeService.java

```diff
@@ -414,6 +432,31 @@ private File saveCConf(CConfiguration cConf, File targetDir) throws IOException
   }
 
   /**
+   * Creates a jar in the given directory that contains a logback.xml loaded from the current ClassLoader.
+   *
+   * @param targetDir directory where the logback.xml should be copied to
+   * @return the {@link File} where the logback.xml jar copied to or {@code null} if "logback.xml" is not found
+   *         in the current ClassLoader.
+   */
+  @Nullable
```

Comment:
At some point we will have to consolidate the common pieces of the mapreduce and spark runners. We can do it then.